OpenRA.Game/Graphics/SpriteRenderable.cs

```diff
@@ -25,23 +25,28 @@ public class SpriteRenderable : IPalettedRenderable, IModifyableRenderable, IFin
 		readonly int zOffset;
 		readonly PaletteReference palette;
 		readonly float scale;
+		readonly WAngle rotation = WAngle.Zero;
 		readonly float3 tint;
 		readonly TintModifiers tintModifiers;
 		readonly float alpha;
 		readonly bool isDecoration;
+		readonly int layer;
```

Comment:
What is `layer`? It doesn't appear to be used and is always set to 0 AFAICT?

Reply:
Ah, this appears to have been a carry-over from my own fork of the engine, where I've added z-ordering of renderables via a 'layer' variable. I may look to add this in a separate PR, as it's useful for overlays in particular, but it has no relevance to the rotation, so I've now removed it.